flowdock-text.js

```diff
@@ -963,25 +963,29 @@ if (typeof FlowdockText === "undefined" || FlowdockText === null) {
     };
   };
 
-  FlowdockText.mentionsAllTags = ["@everyone", "@everybody", "@all", "@anyone", "@anybody"];
```

Comment:
Having them here as well means that when we want to make a change to this list, it needs to be done in (too) many places.